AmplifyPlugins/Auth/AWSAuthPlugin/Support/Utils/AuthUserAttributeKey+Extension.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+// swiftlint:disable cyclomatic_complexity
+extension AuthUserAttributeKey {
```

Comment:
For our own future selves: add a link to the AWS documentation from which these attribute values are taken